<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="IVF111" name="com_Milan_Test.CycleListPageTest" tests="14" failures="1" timestamp="02 Jul 2018 21:40:20 GMT" time="1552.088" errors="0">
  <testcase name="ClickonNewCycleTest" time="78.978" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="CycleListTitleTest" time="74.877" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="NewCycleButtonEnableConditionTest" time="77.190" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="EnabledconditionARTTypeTest" time="197.889" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="ARTTypeOptionTest" time="111.093" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="ProtocolandselectionTest" time="113.873" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="MethodOfSemenCollectionTest" time="108.272" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="SourceofspermPartnerTest" time="110.077" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="SourceofSpermselectionDonorTest" time="112.896" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="SimulationDrugTest" time="111.837" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="LMPDateTest" time="110.580" classname="com_Milan_Test.CycleListPageTest"/>
  <testcase name="SaveTheCycleTest" time="186.938" classname="com_Milan_Test.CycleListPageTest">
    <skipped/>
  </testcase> <!-- SaveTheCycleTest -->
  <testcase name="SaveTheCycleTest" time="77.453" classname="com_Milan_Test.CycleListPageTest">
    <skipped/>
  </testcase> <!-- SaveTheCycleTest -->
  <testcase name="SaveTheCycleTest" time="80.135" classname="com_Milan_Test.CycleListPageTest">
    <failure type="java.lang.AssertionError" message="expected [Record updated successfully.] but found [Cycle is closed,You can not update.]">
      <![CDATA[java.lang.AssertionError: expected [Record updated successfully.] but found [Cycle is closed,You can not update.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com_Milan_Test.CycleListPageTest.SaveTheCycleTest(CycleListPageTest.java:358)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:414)
	at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- SaveTheCycleTest -->
</testsuite> <!-- com_Milan_Test.CycleListPageTest -->
